Link to FeatHub base page added

StephanStS opened the pull request:
**Status**: Ready

**Commit list/description**:
- Additional link to FeatHub may be found easier instead of clicking on one of the software package links.

StephanStS committed:
Link to FeatHub base page added

Additional link to FeatHub may be found easier instead of clicking on one of the software package links.

MichaIng commented:
Good idea. Actually, I am thinking to drop FeatHub. It is not developed anymore for years and often suffers from server down times. All that it offers can be achieved similarly via 👍/👎 reactions in GitHub issues and then sorting them by most 👍. A bid an issue is currently the by times missing references between GitHub issues and FeatHub requests, e.g. users open it at one site but do not open a related one at the other site or do not cross-reference. This increases also maintenance by searching back and forth to collect all info.

Only problem is the migration: We could simply add the current state of votes in the related GitHub issue, but this does not allow sorting them without users giving actual reactions. And there is a large number if requests without attached GitHub issues, and we forgot to mention everywhere that an attached GitHub issue is mandatory to keep discussions and collecting info at a single place rather than two ones.

But this is to think about a later time 😉. Let me just add a note to always open a GitHub issue as well.

MichaIng committed:
v6.34

+ README | Add hint to open a related GitHub issue when creating a FeatHub request
